src/CargoBuild/README.md

```diff
@@ -0,0 +1,121 @@
+### CargoBuild Sdk Prototype
+
+### Project Setup
+To use this sdk you will need the following:
+
+1) in your global.json
+```json
+ "msbuild-sdks": {
+    ...,
+    "Microsoft.Build.CargoBuild": "1.0.270-gf406f8eaa0"
+  },
+```
+
+2) foreach rust project a .cargosproj project file at the same level as your cargo.toml file. The project file should import the CargoBuild sdk.
+```xml
+<Project Sdk="Microsoft.Build.CargoBuild">
+  <PropertyGroup>
+    <TargetFramework> your framework version </TargetFramework>
```

Comment:
Any guidance for what `TargetFramework` to use? It's not clear what the target framework for Rust would be.